test/integration/api-support/test/index.test.js

```diff
@@ -2,7 +2,7 @@
 
 import fs from 'fs-extra'
 import { join } from 'path'
-import AbortController from 'abort-controller'
+import { AbortController } from '@aws-sdk/abort-controller'
```

Comment:
Maybe here we should use the compiled dependency?

Reply:
I'm not sure since the compiled dependency is living in a package path (`next/dist/compiled/@aws-sdk/abort-controller`) and the test is running on the root folder.